service/watcher/configmap/watch.go

```diff
@@ -0,0 +1,153 @@
+package configmap
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/watch"
+
+	"github.com/giantswarm/app-operator/v2/pkg/annotation"
+	applabel "github.com/giantswarm/app-operator/v2/pkg/label"
+)
+
+func (c *AppValue) watch(ctx context.Context) {
+	for {
+		lo := metav1.ListOptions{
+			LabelSelector: applabel.Watching,
+		}
+
+		// Found the highest resourceVersion in cofigMaps CRs
+		cms, err := c.k8sClient.K8sClient().CoreV1().ConfigMaps("").List(ctx, lo)
+		if err != nil {
+			panic(err)
```

Comment:
What happens if this panics? Does the watcher recover?

Reply:
No :( whole gofunctions would quit, I think we need a better error handling in here. Thanks for checking.